plugins/woocommerce/client/admin/client/settings-payments/components/payment-settings-layout/payment-settings-layout.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
This react import is unnecessary. We should type the children prop as React.ReactNode explicitly instead of importing React.
```ts
export const PaymentSettingsLayout = ( { children }: { children: React.ReactNode } ) => (
	<div className="payment-settings-layout">{ children }</div>
);
```